client/index.js

```diff
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import { ApolloClient, InMemoryCache, ApolloProvider } from '@apollo/client';
 import { Provider } from 'react-redux';
-import store from './store';
+import store from './redux/store';
```

Comment:
I like the idea of isolating all the Redux. That's a good move that simplifies the visible files a huge amount and adds some much needed context. Yay.